examples.md

```diff
@@ -1,44 +1,50 @@
 # Examples
 
-- [C# - NuGet](#c---nuget)
-- [Clojure - Lein Deps](#clojure---lein-deps)
-- [D - DUB](#d---dub)
-  - [POSIX](#posix)
-  - [Windows](#windows)
-- [Deno](#deno)
-  - [Linux](#linux)
-  - [macOS](#macos)
-  - [Windows](#windows-1)
-- [Elixir - Mix](#elixir---mix)
-- [Erlang - Rebar3](#erlang--rebar3)
-- [Go - Modules](#go---modules)
-  - [Linux](#linux-1)
-  - [macOS](#macos-1)
-  - [Windows](#windows-2)
-- [Haskell - Cabal](#haskell---cabal)
-- [Haskell - Stack](#haskell---stack)
-- [Java - Gradle](#java---gradle)
-- [Java - Maven](#java---maven)
-- [Node - npm](#node---npm)
-- [Node - Lerna](#node---lerna)
-- [Node - Yarn](#node---yarn)
-- [Node - Yarn 2](#node---yarn-2)
-- [OCaml/Reason - esy](#ocamlreason---esy)
-- [PHP - Composer](#php---composer)
-- [Python - pip](#python---pip)
-  - [Simple example](#simple-example)
-  - [Multiple OS's in a workflow](#multiple-oss-in-a-workflow)
-  - [Multiple OS's in a workflow with a matrix](#multiple-oss-in-a-workflow-with-a-matrix)
-  - [Using pip to get cache location](#using-pip-to-get-cache-location)
-- [Python - pipenv](#python---pipenv)
-- [R - renv](#r---renv)
-- [Ruby - Bundler](#ruby---bundler)
-- [Rust - Cargo](#rust---cargo)
-- [Scala - SBT](#scala---sbt)
-- [Swift, Objective-C - Carthage](#swift-objective-c---carthage)
-- [Swift, Objective-C - CocoaPods](#swift-objective-c---cocoapods)
-- [Swift - Swift Package Manager](#swift---swift-package-manager)
-- [Swift - Mint](#swift---mint)
+- [Examples](#examples)
```

Comment:
This file itself is for examples. Do we really this?

Reply:
This file section an attractive nuisance. I'm pretty sure that VSCode did this on its own.